tools/labs/qemu/Makefile

```diff
@@ -38,6 +43,10 @@ boot: .modinst tap0 tap1 pipe1.in pipe1.out pipe2.in pipe2.out disk1.img disk2.i
 TEMPDIR := $(shell mktemp -u)
 
 $(KCONFIG): qemu/kernel_config.x86
+ifeq ($(ARCH),x86_64)
```

Comment:
This will change the source file, I think it is better to move this block after the copy below and update ^@

Reply:
yeah, that's a great notice, I will fix it.